### PR TITLE
Renovation: Provide only event into onKeyDown handler

### PR DESCRIPTION
**Pull request by LazyLahtak**
Refactor `onKeyDown` event for correct work with Native Angular.
Angular can provide only one argument into event.

### CODE REVIEW

File: js/renovation/button.tsx [hunk 1]
```diff
@@ -208,18 +208,18 @@ export default class Button extends JSXComponent<ButtonProps> {
     useSubmitBehavior && this.submitInputRef.click();
   }
 
-  onWidgetKeyDown(event: Event, options) {
+  onWidgetKeyDown(options) {
     const { onKeyDown } = this.props;
-    const { keyName, which } = options;
+    const { originalEvent, keyName, which } = options;
```
Comment (MaximKudriavtsev) [inline js/renovation/button.tsx:213]: Why we use the name `originalEvent`? Can we call it just `event`?

Comment (LazyLahtak) [reply js/renovation/button.tsx:213]: The `originalEvent` came from old component. But we can rename this.

Comment (LazyLahtak) [reply js/renovation/button.tsx:213]: In fact, `options` - is `event` argument in `onKeyDown(event)`. I think for this reason we have `originalEvent` field.

Comment (MaximKudriavtsev) [reply js/renovation/button.tsx:213]: I see, but in the current context, we don't have any `event` variables and can make our code more readable.

**Review by MaximKudriavtsev — APPROVED** [review]

**Review by pavelgruba — APPROVED** [review]

**Review by churkin — APPROVED** [review]

**Pull request merged**